src/frontend/src/hooks/useWindowSize.js

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from "react";
+
+function useWindowSize() {
+
+    const [windowSize, setWindowSize] = useState({
+      width: undefined,
+      height: undefined,
```

Comment:
Everything seems fine but I would say do not keep the initial values undefined, keep them `window.innerWidth` and `window.innerHeight`